src/containers/BookList.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import Book from "../components/Book";
+import { connect } from "react-redux";
+
+export const BookList = ({ books }) => {
+  console.log("books ", books);
```

Comment:
remember to remove the `console.log` before submitting for review, i know you used it for testing/debugging. 😊